config/lobby/db/003_add_bcrypt_password_column

```diff
@@ -0,0 +1,17 @@
+start transaction;
+
+alter table ta_users add column bcrypt_password character(60) check (char_length(bcrypt_password)=60);
+alter table ta_users alter column password drop not null;
+update ta_users set password=null, bcrypt_password=password where password like '$2a$%';
+alter table ta_users alter column password type character(34);
+alter table ta_users add constraint ta_users_password_check check (char_length(password)=34);
+alter table ta_users add constraint ta_users_check check (password IS NOT NULL OR bcrypt_password IS NOT NULL);
```

Comment:
`ta_users_check` seems a bit generic.  Is there a more descriptive name that we can use?  Maybe `ta_users_require_password_check`?

Reply:
Another suggestion `ta_users_has_password`